definitions/checks/foreman/check_https_proxies.rb

```diff
@@ -0,0 +1,35 @@
+module Checks
+  class CheckHttpsProxies < ForemanMaintain::Check
+    metadata do
+      label :https_proxies
+      for_feature :foreman_database
+      description 'Check for HTTPS proxies from DB'
+      manual_detection
+    end
+
+    def run
+      https_proxies = find_https_proxies
+      unless https_proxies.empty?
+        https_proxy_names = https_proxies.map { |proxy| proxy['name'] }
+        question = "Syncing repositories through an \'HTTP Proxy\' that uses the https:// "\
+                    "protocol is not supported directly with Satellite 6.10.\n"\
+                    "The following proxies use HTTPS: #{https_proxy_names.join(',')}.\n"\
+                    "For a workaround see https://access.redhat.com/solutions/6414991\n"\
+                    'Continue upgrade?'
+        answer = ask_decision(question, actions_msg: 'y(yes), q(quit)')
+        abort! if answer != :yes
+        next_steps.concat([Procedures::KnowledgeBaseArticle.new(:doc => 'use_https_proxy_sat610')])
```

Comment:
Just adding next_steps wont work as I dont see its getting executed. However even if next_steps would execute it means we will show the same message twice if user proceeds with `yes` option?